hw/ip/otbn/dv/uvm/otbn_sim_cfg.hjson

```diff
@@ -53,20 +53,34 @@ name:
   uvm_test: otbn_base_test
   uvm_test_seq: otbn_base_vseq
 
-  // The directory where ELF binaries should be found.
-  //
-  // TODO: This needs deciding upon. Eventually, it might be nice to allow this
-  //       to be overridden on the command line ("I've got a bunch of ELF files
-  //       that stress my new feature; please run the test suite with them")
-  otbn_elf_dir: "{run_dir}/build-out"
+  // When building OTBN tests, we also want to run meson via meson_init.sh to
+  // generate the .env file that describes our toolchain. It will be used as a
+  // pre-run command for tests that need binaries. This step could run either
+  // as a pre-build command or a post-build command. Since it's much quicker
+  // than the SV build, we do it first so that you get a quick result if
+  // something has gone wrong.
+  pre_build_cmds: ["cd {proj_root} && BUILD_ROOT={build_dir} ./meson_init.sh"]
```

Comment:
Hmm, thinking again, this will apply to CSR and other 'common' tests which use a different build mode `cov_reg_top` when coverage is enabled. That is probably ok, but you can do something like this:
```
  build_modes: [
    {
      name: default
      pre_build_cmds: ["cd {proj_root} && BUILD_ROOT={build_dir} ./meson_init.sh"]
    }
  ]
```
This will get it applied to all tests using the `default` build mode. When coverage is enabled, that boils down to ALL tests except all of the common ones.

Reply:
Ah, neat! I'll do that, thanks.